portfolio/src/main/webapp/index.html

```diff
@@ -7,16 +7,55 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is the Moment.</p>
-      <p> My name is Obi Nnaeto and I am a sophomore Informations systems
-          Major at Carnegie Mellon University. I am from Nigeria and I live in New Jersey.
-      </p>
-      <p>Click here to get a quote from my fav tv shows :) :</p>
-      <img src="images/grad_pic.jpeg"/>
+    <div id= "content">
+     <div id = "top_row">
+        <header>
+        <h1> Obi Nnaeto </h1>
+        <div id = "sidebar">
+        <a href = "mailto:obi.nnaeto@gmail.com"><img src = "images/email.png"></a>
+        <a href = "https://www.linkedin.com/in/obi-nnaeto-239933162/"><img src = "images/linkedIn.png"></a>
+        <a href = "https://github.com/cnnaeto21"><img src = "images/git.png"></a>
+        </div>
+        <h3> About Me </h3>
+    </header>
+    <div id = "bio">
+    <p> I am a Sophomore Information Systems Major at Carnegie Mellon University.
+    I am originally from Nigeria and I grew up in Morristown New Jersey I am the middle of 
+    five kids and have no pets. If I could have a pet, it would most be  an aquarium of fish. </p>
+
+    <p>I started coding and working with computers in Highschool. I took the introduction to CS course that
+        my school offered and have been fascinated by the complexity and creativity involved in it ever since.
+        My favorite languages are python and java and the language I hate the most is C. Keep scrolling to learn more about me!
+    </p>
+    </div>
+
+    <div id = "polaroid">
+        <img src = "images/dad_and_I.JPG"/> 
+        <p> My dad and I after my highschool graduation </p>
+        <img src = "images/Academic_Bowl.JPG"/>
+        <p> My teammates and I after winning a bowl game </p> 
+    </div>
+
+    <div id = 'projects'>
+        <h2> Projects </h2>
+        <h4> Here a few projects I have worked on</h4>
+        <ul>
+            <li><a href = 'https://github.com/cnnaeto21/15-112-Term-Project'> Red Light Green Light</a></li>
+            <li><a href = 'https://github.com/cnnaeto21/my-portfolio'> Google SPS Portfolio</a></li>
+      </ul>
+    </div>
+
+      <p>Click here to get a quote from some of my favorite tv shows :) :</p>
```

Comment:
nit: align the div tag appropriately (ie with the same indentation as the closing div tag above)

nit: capitalize TV